rendercv/data/models/rendercv_data_model.py

```diff
@@ -36,6 +37,11 @@ class RenderCVDataModel(RenderCVBaseModelWithoutExtraKeys):
         ),
         validate_default=True,
     )
+    rendercv_settings: Optional[RenderCVSettings] = pydantic.Field(
+        default=None,
+        title="RenderCV Settings",
+        description="The settings of the RenderCV.",
```

Comment:
`validate_default` should be set to `True`.

Reply:
done